### DISCUSSION TITLE
Shortcut duplicates

### DISCUSSION THREAD

**Discussion by tony-sol** [community/community#16580] [Mobile] [answered]
Shortcut "Review requested" is duplicated in shortcuts list if added into panel

Screen record: https://youtu.be/QhQEOwd5buE

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.64.0 (66902451)
Device: iPad Pro 12.9-inch 5 (iOS 15.4.1)
</details>

**Comment by brianlovin** [comment, marked as answer]
Thanks for the recording! I'll share this with the team 😊
